src/include/storage/recovery/recovery_manager.h

```diff
@@ -91,10 +94,10 @@ class RecoveryManager : public common::DedicatedThreadOwner {
   /**
    * Starts a background recovery task. Recovery will fully recover until the log provider stops providing logs.
    */
-  void StartRecovery() {
+  void StartRecovery(bool catalog_only = false) {
```

Comment:
Explanation for the function argument is missing

Reply:
Actually, there should be **no** default argument values in any function. Did clang not catch this and throw an error?